.env

```diff
@@ -2,6 +2,10 @@ DOCKERHUB_NAME=alexithemia
 IMAGE_NAME=docker-kanban
 IMAGE_VERSION=0.0.1
 
+SERVER_PORT=8080
```

Comment:
`.env` file should not be committed. Last ⚠️

Reply:
I think i see why it happened, I'll show you how to remove as I see you added it to `.gitignore` but once committed you have to actually remove it from your repo.

`git rm --cached .env`

This will remove the file from repository tracking, if you don't use the `--cached` flag it will delete the file on disk also.